### DISCUSSION TITLE
Using absolute path as baseTypesPath in near-operation-file

### DISCUSSION THREAD

**Discussion by ASafaeirad** [dotansimha/graphql-code-generator#4924] [Q&A] [answered]
I found a weird behavior in the `near-operation-file` preset.

I use `typescript` and `webpack` path alias in my project. 
in my case, I mapped `@` to `./src`, which allow me to import files like:

```typescript
import query from '@/query/foo.ts'
```

I've tried to reach same behavior in `near-operation-file` but `codegen` converts `baseTypesPath` to relative path

the `codegen` config is:

```yml
  presetConfig:
    extension: .ts
    baseTypesPath: '@/types'
```

which generates:

```ts
import * as Types from './@/types';
```
but I need

```ts
import * as Types from '@/types';
```

Can I force `codegen` to use `baseTypePath` as it is?

**Comment by ASafaeirad** [comment, marked as answer]
I found my answer [here](https://github.com/dotansimha/graphql-code-generator/blob/fd6743d02e7a45b7ccd60f866ea4d88cb18497f5/packages/presets/near-operation-file/src/index.ts#L148) 

It should start with `~` to be considered an absolute path.
```yml
baseTypesPath: '~@/types'
```

But, I can't find that in the documentation.

**Reply by dotansimha** [reply]
Great! 
I created an issue for tracking the missing docs: https://github.com/dotansimha/graphql-code-generator/issues/4935
